include/miral/miral/canonical_window_manager.h

```diff
@@ -55,10 +52,9 @@ class CanonicalWindowManagerPolicy  : public WindowManagementPolicy
     /// Move the child window with the parent
     auto confirm_inherited_move(WindowInfo const& window_info, Displacement movement) -> Rectangle override;
 
-    auto confirm_placement_on_display(
-        WindowInfo const& window_info,
-        MirWindowState new_state,
-        Rectangle const& new_placement) -> Rectangle override;
+    auto confirm_placement_on_display(WindowInfo const& window_info,
+                                      MirWindowState new_state,
+                                      Rectangle const& new_placement) -> Rectangle override;
```

Comment:
Hard pass.

If we can't get clang-format to format these as continuations rather than box-align them then I think we should give up on this; box-align will lead to stupendously long lines.

Reply:
The two options relevant to this block are:
> BinPackParameters: boolean
> If `false`, a function declaration’s or function definition’s parameters will either all be on the same line or will have one line each. 
```
true:
void f(int aaaaaaaaaaaaaaaaaaaa, int aaaaaaaaaaaaaaaaaaaa,
       int aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa) {}

false:
void f(int aaaaaaaaaaaaaaaaaaaa,
       int aaaaaaaaaaaaaaaaaaaa,
       int aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa) {}
```
and

> AllowAllParametersOfDeclarationOnNextLine: boolean
> If the function declaration doesn’t fit on a line, allow putting all parameters of a function declaration onto the next line even if `BinPackParameters` is `false`. 
```
true:
void myFunction(
    int a, int b, int c, int d, int e);

false:
void myFunction(int a,
                int b,
                int c,
                int d,
                int e);
```
Neither matches our preferences.